modules/home/assets/js/components/get-started-section.js

```diff
@@ -0,0 +1,36 @@
+import { Box, Paper, Typography } from '@elementor/ui';
+import List from '@elementor/ui/List';
+import ListItemText from '@elementor/ui/ListItemText';
+import ListItemButton from '@elementor/ui/ListItemButton';
+import Link from '@elementor/ui/Link';
+
+const GetStarted = ( { ...props } ) => {
+	// Will be replaced with a backend solution
+	const adminUrl = elementorAppConfig.admin_url;
+	return (
+		<Paper elevation={ 0 } sx={ { p: 3, display: 'flex', flexDirection: 'column', gap: 2 } }>
+			<Typography variant="h6">{ props.getStartedData.header.title }</Typography>
```

Comment:
`props.getStartedData.header.title`
Should we maybe validate that the nested object is valid and all properties are accessible to avoid error, or is it already validated somewhere else? It might also be possible for it to always have needed values and them my question is irrelevant :)